modules/test/playwright/tests/dynamic-data-mapping-form-web/main/formSubmission.spec.ts

```diff
@@ -169,4 +170,48 @@ test.describe('Manage forms through submission page', () => {
 
 		await expect(page.getByText('123456')).toBeVisible();
 	});
+
+	test('verify that a Form can require CAPTCHA before being accessed', async ({
+		formBuilderPage,
+		formsPage,
+		page,
+	}) => {
+		await formsPage.goTo();
+
+		await test.step('create a form containing a text field and CAPTCHA validation', async () => {
+			await formsPage.newFormButton.first().click();
+
+			await formBuilderPage.fillFormTitle('Form' + getRandomInt());
+
+			await page
+				.getByRole('button', {name: 'Text field'})
+				.first()
+				.dblclick();
+
+			await page.getByRole('button', {name: 'settings'}).click();
+
+			await page.getByLabel('Require CAPTCHA').click();
+
+			await page.getByRole('button', {name: 'Done'}).click();
```

Comment:
Hi @DavysonMelo, could you use `formSettingsDoneButton` here?

By using these locators, we're reducing the maintenance effort of tests when we need to modify these components in the future.